### PR TITLE
Release notes for 1.19.6

### PR DESCRIPTION
**Pull request by ksubrmnn**
## Description

Release notes for 1.19.6

## Reviewers

- [ ] Ambient
- [ ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [X] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [ ] Localization/Translation

### CODE REVIEW

File: content/en/news/releases/1.19.x/announcing-1.19.6/index.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+---
+
+This release note describes what’s different between Istio 1.19.5 and 1.19.5.
+
+{{< relnote >}}
+
+## Changes
+
+- **Fixed** a bug where overlapping wildcard hosts in a VirtualService would produce incorrect routing configuration when wildcard services were selected (e.g. in ServiceEntries).
+  ([Issue #45415](https://github.com/istio/istio/issues/45415))
```
Comment (dhawton) [inline content/en/news/releases/1.19.x/announcing-1.19.6/index.md:17]:
```suggestion
- **Fixed** a bug where overlapping wildcard hosts in a `VirtualService` produces incorrect routing configurations
  when wildcard services were selected (e.g. in `ServiceEntry`).
  ([Issue #45415](https://github.com/istio/istio/issues/45415))
```

File: content/en/news/releases/1.19.x/announcing-1.19.6/index.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+- **Fixed** an issue where the Istio injection webhook may be modified in dry-run mode.
+  ([Issue #48241](https://github.com/istio/istio/issues/48241))
+
+- **Fixed** an issue where the QUIC listeners were not correctly created when dual-stack is enabled.
+  ([Issue #48336](https://github.com/istio/istio/issues/48336))
+
+- **Fixed** an issue `proxyconfig ecds` didn't show all `EcdsConfigDump`.
```
Comment (dhawton) [inline content/en/news/releases/1.19.x/announcing-1.19.6/index.md:25]:
```suggestion
- **Fixed** an issue where `istioctl proxy-config ecds` didn't display all `EcdsConfigDump`.
```

File: content/en/news/releases/1.19.x/announcing-1.19.6/index.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+- **Fixed** an issue where new endpoints may not be sent to proxies.
+  ([Issue #48373](https://github.com/istio/istio/issues/48373))
+
+- **Fixed** an issue where installing with Stackdriver and custom configurations would lead to Stackdriver not being enabled.
```
Comment (dhawton) [inline content/en/news/releases/1.19.x/announcing-1.19.6/index.md:30]:
```suggestion
- **Fixed** an issue where installing with Stackdriver and using custom configurations would prevent Stackdriver from being
  enabled.
```

**Review by hanxiaop — APPROVED** [review]
LGTM

**Comment by ericvn** [comment]
LGTM. Will let @dhawton review again for earlier comments.

**Review by dhawton — APPROVED** [review]
LGTM as well

**Pull request merged**